src/inc-lib-init.php

```diff
@@ -1084,336 +1084,525 @@ function lovd_getInstallURL ($bFull = true)
+                //  If the '*' is given, the DNA must be of type coding (prefix = c) or non-coding (prefix = n).
+                if ($bCheckHGVS) {
+                    return false;
+                }
+                $aResponse['errors']['EFALSEUTR'] =
+                    'Only variants of type coding (c) or non-coding (n) can hold \'*\'s in their positions.';
+                return $aResponse; // Fixme; take another look at this.
             }
-            if (strpos($sVariant, '>') !== false && $sEndPosition) {
-                // Substitutions are not allowed to have a range.
+            if ($sTranscriptID === '') {
+                // If the '*' symbol is given, we must also have a transcript.
+                // This mistake does not lie with the user, but with LOVD as the function would not have
+                //  been called with a transcriptID or transcriptID=false.
                 return false;
             }
+            // We add the length of the transcript to the position if a '*' has been found.
+            $aVariant[$sPosition] = (int)substr($aVariant[$sPosition], 1) + $aTranscriptOffsets[$sTranscriptID];
 
-            if ($sSuffix) {
-                // Suffix not allowed in some cases.
-                if (strpos($sVariant, '>') !== false || $sVariant == 'inv' || substr($sVariant, 0, 1) == '|' || $sVariant == '=') {
-                    // No suffix allowed for substitutions, inversions, methylation or WT calls.
-                    return false;
-                } elseif (in_array($sVariant, array('del', 'dup')) && !preg_match('/^[ACTG]+$/', $sSuffix)) {
-                    // Only allow bases as suffix for deletions and duplications.
-                    return false;
-                } elseif ($sVariant == 'delins'
-                    && !preg_match('/^([ACTG]+|\([0-9]+\))$/', $sSuffix)
-                    && !preg_match('/^([0-9]+|[0-9]+[+-][0-9]+)_([0-9]+|[0-9]+[+-][0-9]+)$/', $sSuffix)
-                    && !preg_match('/^\[[NX][CMR]_[0-9]{6,9}\.[0-9]+:[cgmn]\.([0-9]+|[0-9]+[+-][0-9]+)_([0-9]+|[0-9]+[+-][0-9]+)\]$/', $sSuffix)) {
-                    // Only allow bases, length, or positions as suffix for deletion-insertion events.
-                    // Position ranges for deletion-insertions used to be conversions.
-                    return false;
-                } elseif ($sVariant == 'ins' && !preg_match('/^([ACTG]+|\([0-9]+\)|[0-9]+_[0-9]+|\[NC_[0-9]{6}\.[0-9]+:[0-9]+_[0-9]+\])$/', $sSuffix)) {
-                    // Supported are insertions with bases, length, or with position fields.
-                    return false;
+        } elseif ($aVariant[$sPosition] == '?') {
+            $aResponse['messages']['IUNKNOWNPOSITIONS'] = 'This variant contains unknown positions.';
+            
+        } else {
+            // When no '*' is found, we can safely cast the position to integer.
+            $aVariant[$sPosition] = (int)$aVariant[$sPosition];
```

Comment:
What is the advantage of casting to an int? I don't think there is any, right?
We used to do this for the intronic positions to get rid of the '+', but these are not intronic positions, as far as I can see. It's OK to leave it, but perhaps then add a comment that this isn't required functionality.